### ISSUE TITLE
create_trade errors when not 18 decimals

### ISSUE THREAD

**Issue by jaggedsoft** [sammchardy/python-idex#35]
Bug: create_trade is not working for coins like LTO, WMB, EPY that have less than 18 decimals.

```py
from idex.client import Client
client = Client("redacted", "redacted")
depth = client.get_order_book(market, 1)
orderHash = depth["asks"][0]["orderHash"]
amount = depth["asks"][0]["amount"]

order = client.create_trade(orderHash, "WMB", amount)
print(order)
```
![image](https://user-images.githubusercontent.com/4283360/52640766-938f5000-2ed7-11e9-9867-e8880e01df38.png)
**Error 504: Gateway Timeout**

**Comment by jaggedsoft** [comment]
I believe this is an issue with IDEX. I am trying to post a message manually to the endpoint, with the same result

**Comment by jaggedsoft** [comment]
Reopening this because IDEX has been less than helpful in regards to fixing the issue. I have the same issue when I post the transactions manually using javascript/web3, which is why I believe the problem is not related to your library and is entirely on IDEX's end.

![unknown](https://user-images.githubusercontent.com/4283360/53174812-f759fd00-35ea-11e9-9d40-81035ea19d4d.png)

In this list, I have tried to market buy 0.051 ETH from every pair. There's 147 pairs with less than 8 decimals of precision on IDEX, and only 12 of them have more than 10+ eth volume, suggesting more people than just myself and my friends are affected by this. I also tried making a brand new account and it has the same problem.
